public/TCP/TCPClient.js

```diff
@@ -10,34 +17,46 @@ const limitAttempts = 3;
 function getConnectedClient() {
   //console.log('Attempting to create TCP client and connect to server..');
   const client = new net.Socket();
+  messageProtocol = global.settings.messageProtocol;
+  console.log(messageProtocol);
 
   client.connect({
     port: global.settings.port,
     host: global.settings.host,
   });
 
   client.on('connect', function() {
-    sendData(client, {
-      surge: 0.0,
-      sway: 0.0,
-      heave: 0.0,
-      roll: 0.0,
-      pitch: 0.0,
-      yaw: 0.0,
-      autodepth: false,
-      autoheading: false,
-    });
+    console.log(`Client: connection established with server!`);
+    console.log(messageProtocol);
```

Comment:
Kommenter ut logs, det fucker med testene og brukes ikke i prod

Reply:
Line 29 could be nice to have, as it is visible in the terminal when starting the server.
Removed line 30 👌🏼